app/main.py

```diff
@@ -2,0 +1,21 @@
+from typing import Union
+
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> Union[str, None]:
+    masks_needed = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_needed += 1
+
+    if masks_needed:
+        return f"Friends should buy {masks_needed} masks"
+
+    return f"Friends can go to {cafe.name}"
```

Comment:
Code Style: It's good practice to use f-strings for string formatting in Python. It makes your code more readable. You are already using it correctly in this line. Keep it up!